tutorials/quickstart/quickstart-tutorial-intro-paragraph.include.md

```diff
@@ -1 +1,5 @@
-Learn why software systems built on asynchronous messaging using NServiceBus are superior to traditional synchronous HTTP-based web services. We'll also show how NServiceBus guarantees reliability and extensibility that can't be achieved with REST.
+It is **very difficult** to build a distributed software system correctly from scratch. Would you rather read all 736 pages of the [Enterprise Integration Patterns](https://www.enterpriseintegrationpatterns.com/) book and then spent months creating, testing, and documenting a communication framework so that different services can talk to each other? Or would you rather use a framework that builds on well-defined distributed system design principles and guides you toward the "pit of success?"
```

Comment:
Personal preference: Not a fan of rhetorical questions though I can't say why. An alternative is to you use:

> You _could_ read all 736 pages of ..... Or instead, you could use a framework that builds....

I'm also leery about saying using NServiceBus means you don't have to read a book that is well-regarded in the industry, even though no one will claim it's riveting reading.